src/Workspaces/VisualBasic/Portable/CodeCleanup/Providers/AddMissingTokensCodeCleanupProvider.vb

```diff
@@ -508,16 +511,26 @@ Namespace Microsoft.CodeAnalysis.CodeCleanup.Providers
                             End If
 
                         Else
-                            Dim ifDirective = TryCast(originalParent, IfDirectiveTriviaSyntax)
-                            If ifDirective IsNot Nothing Then
-                                Dim previousToken = ifDirective.Condition.GetLastToken(includeZeroWidth:=True)
-                                Dim nextToken = ifDirective.GetLastToken.GetNextToken
+                            Dim regularElseIf = TryCast(originalParent, ElseIfStatementSyntax)
+                            If regularElseIf IsNot Nothing Then
+                                Dim previousToken = regularElseIf.Condition.GetLastToken(includeZeroWidth:=True)
+                                Dim nextToken = regularElseIf.GetLastToken.GetNextToken
```

Comment:
I see that you're just doing the same thing that was already there, but why does `previousToken` use `includeZeroWidth`, but `nextToken` not?  @heejaechang any ideas?